mmocr/models/textdet/losses/fce_loss.py

```diff
@@ -0,0 +1,166 @@
+import torch
+import numpy as np
+import torch.nn.functional as F
+from torch import nn
+
+from mmdet.core import multi_apply
+from mmdet.models.builder import LOSSES
+
+
+@LOSSES.register_module()
+class FCELoss(nn.Module):
+    """The class for implementing FCENet loss
+    FCENet(CVPR2021): Fourier Contour Embedding for Arbitrary-shaped Text Detection
+
+    [https://arxiv.org/abs/2104.10442]
+    """
+
+    def __init__(self, fourier_degree, sample_points, ohem_ratio=3.):
+        """Initialization.
+
+        Args:
+            fourier_degree (int) : The maximum fourier transform degree k.
+            sample_points (int) : The sampling points number of regression loss. If this Arg is too small, fcenet is
+                                tend to be overfitting.
+            ohem_ratio (float): the negative/positive ratio in OHEM.
```

Comment:
```suggestion
            fourier_degree (int) : The maximum Fourier transform degree k.
            sample_points (int) : The sampling points number of regression loss. If it is too small, fcenet
                                tends to be overfitting.
            ohem_ratio (float): the negative/positive ratio in OHEM.
```

The fourier_degree appears in many classes, such as fceloss, fcenet and fcehead.
suggest to initialize it in one class and then pass it to other classes.
In this way, it can avoid mismatching due to multiple initialization at multiple classes.